plugins/inputs/smart/README.md

```diff
@@ -112,6 +112,13 @@ smartctl --scan -d nvme
   
     ## Timeout for the cli command to complete.
     # timeout = "30s"
+    
+    ## Optionally call smartctl and nvme-cli sequentially to gather disk attributes.
+    ## By default, smartctl and nvme-cli are called in separate threads (goroutines) to gather disk attributes.
+    ## Some devices (e.g. disks in RAID arrays) may have access limitations that require sequential reading of
+    ## SMART data - one individual array drive at the time. In such case please set this configuration option to false
+    ## to get readings for all drives.
+    # parallel_reads = true
```

Comment:
I expect users will want other methods to gather attributes besides completely parallel and completely serial. For example, it might be a good idea on a machine with multiple HBAs to gather each HBA in parallel, each with its own thread.

Since it's possible/likely we'll have other methods, I think we should use a string setting to select the method. That would allow us to add another read method without adding another variable.

I'd also like to avoid the terms 'serial' and 'parallel' here because users might think we're talking about SATA and PATA.

for example:

```toml
    ## valid options: concurrent, sequential
    # read_method = "concurrent"
```

With a string setting like this example, we could add "concurrent_hba" or whatever we want in the future.

Reply:
@reimda thanks for paying attention to it. I have updated the code to handle string-type `read_method` config value instead of boolean. Please check out the new changes.